templates/deno/.vscode/resolve_npm_imports.json

```diff
@@ -5,11 +5,13 @@
   "// Deno-only dependencies may be imported via URL imports (without using import maps).": "",
   "imports": {
     "// `@remix-run/deno` code is already a Deno module, so just get types for it directly from `node_modules/`": "",
-    "@remix-run/deno": "https://esm.sh/@remix-run/deno@1.6.0",
+    "@remix-run/deno": "https://esm.sh/@remix-run/deno@1.6.0/index.ts",
+    "@remix-run/server-runtime": "https://esm.sh/@remix-run/server-runtime@1.6.0",
     "@remix-run/dev/server-build": "https://esm.sh/@remix-run/dev@1.6.0/server-build",
     "@remix-run/react": "https://esm.sh/@remix-run/react@1.6.0",
     "react": "https://esm.sh/react@17.0.2",
     "react-dom": "https://esm.sh/react-dom@17.0.2",
-    "react-dom/server": "https://esm.sh/react-dom@17.0.2/server"
+    "react-dom/server": "https://esm.sh/react-dom@17.0.2/server",
+    "mime": "https://esm.sh/mime@3.0.0"
```

Comment:
Let's keep these alphabetical, so it's easier to find the correct ones if we ever have a long list here